src/custom/CortiAuth.ts

```diff
@@ -116,21 +135,46 @@ export class Auth extends FernAuth {
         }, requestOptions));
     }
 
+    /**
+     * Patch: PKCE-specific method for Authorization code flow
+     */
+    public getCodePkceFlowToken(
+        request: AuthorizationCodePkceServer,
+        requestOptions?: FernAuth.RequestOptions,
+    ): core.HttpResponsePromise<Corti.GetTokenResponse> {
+        return core.HttpResponsePromise.fromPromise(this.__getToken_custom({
+            ...request,
+            grantType: "authorization_code",
+        }, requestOptions));
+    }
+
     /**
      * Patch: copy of this.__getToken with patches
      */
     private async __getToken_custom(
         /**
          * Patch: added additional fields to request to support Authorization code flow
          */
-        request: Corti.AuthGetTokenRequest & Partial<{
+        request: (Corti.AuthGetTokenRequest | AuthorizationCodeServer | AuthorizationCodePkceServer | AuthorizationRefreshServer) & Partial<{
             grantType: "client_credentials" | "authorization_code" | "refresh_token";
             code: string;
             redirectUri: string;
+            codeVerifier: string;
             refreshToken: string;
         }>,
         requestOptions?: FernAuth.RequestOptions,
     ): Promise<core.WithRawResponse<Corti.GetTokenResponse>> {
+        const grantType = (request as any).grantType || "client_credentials";
```

Comment:
Using `any` is a very bad practice in TypeScript, we try to avoid it at all costs